cdp-agentkit-core/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## [0.0.5] - 2024-11-15
+
+### Added
+
 - Added `account_mentions` action to `twitter-langchain`.
```

Comment:
This package is a dependency of `twitter-langchain`? Should the context of this changelog just reflect that the actions are added to `cdp-agentkit-core`?

Reply:
Side-note: But feels weird to have twitter/X dependencies in our "core" library 🤔

Reply:
yeah agreed i updated the changelog! and yeah i think our actions can be split into multiple libs over time if needed